x-pack/plugins/cases/server/services/cases/transform.ts

```diff
@@ -129,7 +129,7 @@ export function transformAttributesToESModel(caseAttributes: Partial<CaseTransfo
 
   return {
     attributes: {
-      ...restAttributes,
+      ...mapNonTransformedSOAttributes(restAttributes),
```

Comment:
nit: why `NonTransformedAttributes`? Could be `map remaining attributes` or something